es/contract/compiler.js

```diff
@@ -48,6 +48,10 @@ async function contractGetACI (code, options = {}) {
   return this.http.post('/aci', { code, options }, options)
 }
 
+async function setCompilerUrl (url) {
```

Comment:
@nduchak Does this function need to be declared as `async` even if it doesn't return anything?

Reply:
Yes, you are right.

Reply:
Fixed